draft/src/main.rs

```diff
@@ -24,15 +26,50 @@ fn main() {
     let draft_info = create_draft_info(&list_filename);
     let mut draft_packs = create_draft_packs(&draft_info);
 
-    println!("draft packs: {:?}", draft_packs);
+    let mut drafters: Vec<Box<Drafter>> = vec![];
+    for _i in 0..draft_info.num_drafters {
+        drafters.push(Box::new(random_drafter::RandomDrafter {}));
+    }
+
+    let mut draft_state = DraftState::new(drafters, draft_packs);
+    println!("draft state: {:?}", draft_state);
 
-    // TODO: Implement full draft of random drafters
-    let drafter = random_drafter::RandomDrafter {};
+    run_draft(&draft_info, &mut draft_state);
+}
 
-    let pack = &draft_packs[0][0];
-    let cards_owned : Vec<Card> = vec![];
+fn run_draft(draft_info: &DraftInfo, draft_state: &mut DraftState) {
```

Comment:
I think this function could use a couple comments, especially to help call out what it's doing with index calculation.

Reply:
Done.